packages/apollo-gateway/src/index.ts

```diff
@@ -236,6 +236,8 @@ export class ApolloGateway implements GraphQLService {
 
   public async load(options?: { engine?: GraphQLServiceEngineConfig }) {
     await this.updateComposition(options);
+    const graphId = options && options.engine && options.engine.graphId;
+    this.logger.info(`Gateway loaded schema${graphId && ` for service ${graphId}`}.`)
```

Comment:
probably want a variant in there? And maybe the method it used! I've often been uncertain from my logs if I'm running in managed or unmanaged mode :)

Reply:
👍 resolved in eded62a